python/tk_houdini_renderman/handler.py

```diff
@@ -77,65 +65,61 @@ def submit_to_farm(self, node, network):
         global submission
         # Start submission panel
         submission = FarmSubmission(
-            self.app, node, file_name, "50", framerange, network=network
+            self.app, node, file_name, 50, framerange, render_paths, network=network
         )
         submission.show()
 
     def execute_render(self, node, network):
-        # Validate node
-        if not self.__validate_node(node, network):
-            return
-
-        # Prepare node and set all parameters
-        render_path = self.__calculate_path(node, network)
-        self.__apply_path(node, render_path, network)
-        self.__create_directory(render_path)
-
-        # Set filters
-        if network == "lop":
-            filters = self.__check_lop_filters(node)
-            self.__set_lop_filter_filename(node, filters)
-
-        if network == "rop":
-            filters = self.__check_rop_filters(node)
-            self.__set_rop_filter_filename(filters, render_node=node)
+        # Create directories
+        render_paths = node.type().hdaModule().get_render_paths()
+        for p in render_paths:
+            self.__create_directory(p)
+
+        # # Set filters
```

Comment:
Should this be removed?

Reply:
Kept this in case it was needed for the coming lop version of the node, but I could just as well look back on GitHub. Will check for other currently redundant code as well.